client/app/hosting/module/add/hosting-module-add.html

```diff
@@ -182,12 +182,12 @@
                                data-i18n-static="hosting_configuration_tab_modules_create_step2_db_passwd"></label>
                         <div class="col-md-6">
                             <input type="password" autocomplete="off" class="form-control" id="dbPasswd" name="dbPasswd" required
-                                   data-ng-minlength="8"
-                                   data-ng-maxlength="model.databaseSelected.name ? 12 : 31"
+                                   data-ng-minlength="model.minDBPasswordLength"
```

Comment:
`Database` instead of `DB` is more readable.

Reply:
We have both in this file `formModuleDB.dbPasswd` and `databaseHostedSelected`...
But ok I will change it.